.github/workflows/ci.yml

```diff
@@ -58,42 +58,48 @@ jobs:
           - os: windows-arm64
             name: Windows aarch64
     env:
+      FORCE_COLOR: 1
       SCIENCE_AUTH_API_GITHUB_COM_BEARER: ${{ secrets.GITHUB_TOKEN }}
     steps:
-      - name: Setup Python 3.12
+      - name: Install the latest version of uv
         if: matrix.docker-platform == '' && matrix.os != 'windows-arm64'
-        uses: actions/setup-python@v5
-        with:
-          python-version: 3.12
-      - name: Setup Python 3.12
+        uses: astral-sh/setup-uv@v4
+      - name: Install the latest version of uv
         if: matrix.os == 'windows-arm64'
+        shell: cmd
         run: |
-          py -3.12 -m venv .venv
-          echo "$(pwd)/.venv/Scripts" >> "${GITHUB_PATH}"
-      - name: Setup Nox
-        if: matrix.docker-platform == ''
-        run: pip install nox
+          "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\Tools\VsDevCmd.bat"
+          cargo install --git https://github.com/astral-sh/uv uv
+          uv -V
+      - name: Setup x86_64 Python for Prism
+        if: matrix.os == 'windows-arm64'
+        run: |
+          UV_PYTHON_VERSION=cpython-3.12.8-windows-x86_64-none
```

Comment:
Likewise, PBS has no Windows ARM release; so this gets uv using x86-64 PBS / Windows PRISM emulation. This was done similarly, but with more fuss in noxfile.py.